persistence/sql/timeouts_connection_sqlpersistence_[,7).partial.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Timeouts Persister
+component: SqlPersistence
+reviewed: 2020-02-27
+---
+
+## Connection
+
+The timeouts persister can be configured to use a dedicated connection builder. For example, it may be used for creating timeouts tables in a separate database.
+
+snippet: SqlPersistenceTimeoutConnection
+
+## Migrating timeouts
```

Comment:
This section is duplicated in the main document.

Reply:
updated